.github/workflows/preview-web.yaml

```
@@ -26,6 +26,14 @@ jobs:
         env:
           CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
 
+  Analyze bundle sizes:
+    uses: transferwise/actions-next-bundle-analyzer@master
+    with:
+      base-branch: main
+      working-directory: /apps/web
+    env:
```

actionlint (1.7.7)

when a reusable workflow is called with "uses", "env" is not available. only following keys are allowed: "name", "uses", "with", "secrets", "needs", "if", and "permissions" in job "Analyze bundle sizes"

(syntax-check)